rfhub2/db/repository/keyword_repository.py

```diff
@@ -146,3 +148,8 @@ def get_with_stats(self, item_id: int) -> Optional[KeywordWithStats]:
         result = self._items_with_stats.filter(self._id_filter(item_id)).first()
         if result:
             return self.from_stats_row(result)
+
+    def update(self, item: Keyword, update_data: dict) -> Keyword:
+        if 'tags' in update_data:
+            update_data['tags'] = Keyword.from_json_list(update_data['tags'])
+        super().update(item, update_data)
```

Comment:
`return` is required to return Keyword

Reply:
like `return super().update(item, update_data)` ?